packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -159,6 +163,17 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
             events: [ 'mousedown' ],
         };
         // width={this.props.style.chartWidth}
+        if (this.state.outputStatus === ResponseStatus.COMPLETED && this.state.xyTree.length === 0 ) {
+            return <React.Fragment>
+                {
+                    <p className='no-data' style={{fontSize: 20, marginRight: '5px', marginLeft: '5px', justifyContent:'center', alignItems:'center'}}>
```

Comment:
Why are there style properties here that contradict or seem to repeat the styles in the no-data CSS class (see below: output-components-style.css line 84)? Why not just have all the style properties in the CSS class?

Examples:
* fontSize: 20 vs. font-size: 24
* justifyContent: center, alignItems: center vs. text-align: center